api/v2/v2.go

```diff
@@ -194,35 +199,50 @@ type VerifyBatch struct {
 	Timestamps []int64  `json:"timestamps"`
 }
 
-// VerifyBatchReply is returned by the server with the status results for the requested
-// digests and timestamps.
+// VerifyBatchReply is returned by the server with the status results for the
+// requested digests and timestamps.
 type VerifyBatchReply struct {
 	ID         string            `json:"id"`
 	Digests    []VerifyDigest    `json:"digests"`
 	Timestamps []VerifyTimestamp `json:"timestamps"`
 }
 
-// ChainInformation is returned by the server on a verify digest request. It contains
-// the merkle path of that digest.
+// ChainInformation is returned by the server on a verify digest request.
+// It contains the merkle path of that digest.
 type ChainInformation struct {
 	ChainTimestamp int64         `json:"chaintimestamp"`
 	Transaction    string        `json:"transaction"`
 	MerkleRoot     string        `json:"merkleroot"`
 	MerklePath     merkle.Branch `json:"merklepath"`
 }
 
-// CollectionInformation is returned by the server on a verify timestamp request.
-// It contains all digests grouped on the collection of the requested block timestamp.
+// CollectionInformation is returned by the server on a verify timestamp
+// request. It contains all digests grouped on the collection of the
+// requested block timestamp.
 type CollectionInformation struct {
 	ChainTimestamp int64    `json:"chaintimestamp"`
 	Transaction    string   `json:"transaction"`
 	MerkleRoot     string   `json:"merkleroot"`
 	Digests        []string `json:"digests"`
 }
 
-// WalletBalanceReply returns balance information of the decred wallet.
+// WalletBalanceReply is returned by server on a balance information of the
+// decred wallet.
 type WalletBalanceReply struct {
 	Total       int64 `json:"total"`
 	Spendable   int64 `json:"spendable"`
 	Unconfirmed int64 `json:"unconfirmed"`
 }
+
+// LastAnchorReply is returned by server on a last succcessful anchor info
+// request, last successful anchor is the last dir which was anchored on the
```

Comment:
`last successful anchor is the last dir which was anchored on the`

I don't know what this means.  This is too specific.  You should not be including implementation details, in this case the filesystem implementation details, in the api comments.  The implementation details may change.  The API will not. This needs comment needs to relay the information that the reply will contain the last anchor tx that was included in a block and that the `ChainTimestamp` field will not be populated until the block has 6 confirmations.  Nothing about the filesystem.